### PR TITLE
fix(ReportDownload): RHICOMPL-2514 - Fix issues with unsupported systems only

### PR DESCRIPTION
**Pull request by bastilian**
## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [x] General Coding Practices

### CODE REVIEW

**Review by skateman — APPROVED** [review]
The Seal of Approval

File: src/SmartComponents/ReportDownload/Components/ReportPDF.js [hunk 1]
```diff
@@ -17,6 +17,13 @@ import PanelItem from './ReportPDF/PanelItem';
 import SubSection from './ReportPDF/SubSection';
 import ComplianceChart from './ReportPDF/ComplianceChart';
 
+const calculatePercent = (compliantHostCount, testResultHostCount) => {
+  const calculatedPercent = Math.floor(
+    100 * (compliantHostCount / testResultHostCount)
+  );
+  return isNaN(calculatedPercent) ? 0 : calculatedPercent;
```
Comment (vkrizan) [inline src/SmartComponents/ReportDownload/Components/ReportPDF.js:24]: Would be "safer" to check for division by zero?

Comment (bastilian) [reply src/SmartComponents/ReportDownload/Components/ReportPDF.js:24]: ... Maybe even more elegant. I'll rewrite that bit.

Comment (bastilian) [reply src/SmartComponents/ReportDownload/Components/ReportPDF.js]: Updated.

**Review by vkrizan — APPROVED** [review]

**Pull request merged**